features/frameworks/greentea-client/source/greentea_test_env.cpp

```diff
@@ -67,7 +67,9 @@ static void greentea_write_string(const char *str);
  *           This function is blocking.
  */
 void _GREENTEA_SETUP_COMMON(const int timeout, const char *host_test_name, char *buffer, size_t size) {
+#if defined(MBED_CONF_RTOS_PRESENT)	
```

Comment:
We should be able to get some metrics in baremetal, for example CPU stats. Rather than disabling the call to `greentea_metrics_setup` disable the metrics that are not available withour an RTOS in `greentea_metrics_setup`.

Reply:
Yes Done